v2/emailpassword/common-customizations/sessions/session-verification-in-api/get-session.mdx

```diff
@@ -0,0 +1,96 @@
+---
+id: get-session
+title: Using the getSession function
+hide_title: true
+---
+
+
+<!-- COPY DOCS -->
+<!-- ./session/common-customizations/sessions/session-verification-in-api/get-session.mdx -->
+
+import BackendSDKTabs from "/src/components/tabs/BackendSDKTabs"
+import TabItem from '@theme/TabItem';
+
+
+// add try catch, add all possible errors and in comments decribe how to handle them, also add the other two ways of doing session verification.
+
+# Session Verification using `getSession`
+
+If you need to verify a session in your API, you should be using the [verifySession](./verify-session) middleware whenever possible, but, if you need session verification in the middle of your API logic, then use `getSession` as shown below.
+
+## Requiring an active session
+
+<BackendSDKTabs>
+<TabItem value="nodejs">
+
+```jsx
+let Session = require("supertokens-node/recipe/session");
+
+// highlight-start
+app.post("/like-comment", (req, res) => {
+    
+    // this will contain the session object post verification
+    let session
+
+    try {
+        session = await Session.getSession(req, res);
+    } catch (err) {
```

Comment:
- get rid of the try catch
- don't you need to check for `session === undefined` below?